docs/modules/operation/pages/service-assurance/polling-packages.adoc

```diff
@@ -152,80 +151,80 @@ The following example shows which configuration is applied for a specific servic
   <downtime interval="600000" begin="43200000" end="432000000" />
 </package>
 ----
-<1> Polling interval in the packages are 300 seconds and 30 seconds
+<1> Polling interval in the packages are 300 seconds and 30 seconds.
 <2> Different polling interval for the ICMP service
 <3> Different retry settings for the ICMP service
 <4> Different timeout settings for the ICMP service
```

Comment:
```suggestion
<2> Different polling interval for the ICMP service.
<3> Different retry settings for the ICMP service.
<4> Different timeout settings for the ICMP service.
```

Reply:
While I dislike having some items with periods and other ones without, since there is no verb here, they don't need periods.

Reply:
Good catch. I double checked the NH style guide, and they do recommend all periods, when list is mix of sentences and sentence fragments. I've been using them only with full sentences, and I hate the way that looks!